BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/rest/service/PendingSubmissionUtil.java

```diff
@@ -0,0 +1,128 @@
+package uk.ac.ebi.biostd.webapp.application.rest.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionDto;
+import uk.ac.ebi.biostd.webapp.application.rest.dto.PendingSubmissionListItemDto;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+@Component
+@Slf4j
+@AllArgsConstructor
+public class PendingSubmissionUtil {
+
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+    private static final String ACCNO_PREFIX = "TMP_";
+
+    private final ObjectMapper objectMapper;
+
+    public String asString(PendingSubmissionDto dto) {
+        return objectMapper.valueToTree(dto).toString();
+    }
+
+    public Optional<PendingSubmissionDto> parse(String data) {
```

Comment:
this name and other looks to generic for me so when used in lambda is not clear their purpose, for example this can be changed to something like `asSubmissionDto` in general my opinion is that api user is more interested in what method do (convert to dto) that  how does it do it (parsing a string)

Reply:
Agree. Done.